internal/kibana/space.go

```diff
@@ -61,6 +61,12 @@ func ResourceSpace() *schema.Resource {
 			Optional:     true,
 			ValidateFunc: validation.StringMatch(regexp.MustCompile("^data:image/"), "must be a valid data-URL encoded image"),
 		},
+		"solution": {
```

Comment:
This field is only supported from Elastic Stack 8.18.0 and higher. You'll need to check that the provider is running against a supported server version. You can check `internal/kibana/slo.go` line 701-709 for an example of how to do this.